cli/dts/lib.deno.unstable.d.ts

```diff
@@ -901,6 +901,57 @@ declare namespace Deno {
    */
   export function shutdown(rid: number, how: ShutdownMode): Promise<void>;
 
+  /** The type of the resource record.
+   * Only the listed types are supported currently. */
+  export type RecordType =
```

Comment:
Easy to add later, but many people doing DNS lookups directly will want TXT and SRV records too.

Reply:
@magurotuna let's address that before landing

Reply:
@mrkurt 
`TXT` and `SRV` cannot easily be transferred from Rust to JS due to a quirk of `trust_dns` crate. 

As far as I understand, the content of `TXT` record can not always be represented as Rust's string. So I think that if we send it to JS, it will become `Uint8Array` in JS. Will it be fine for those who want DNS lookups?

And, as of `SRV`, what format would users find useful? One simple format is something like `<priority> <weight> <port> <target>` (separated by space), do you think this is good?

Reply:
I would rather have structured data for SRV.

Reply:
@lucacasonato I see, so the following should be better

```ts
{
  priority: number;
  weight: number;
  port: number;
  target: string;
}
```

Reply:
Yeah - same goes for `MX` records in the future.

Reply:
It shouldn't be so hard to add `MX` support in this PR, so I'll do that 👍